pkg/worker/digest_email_job.go

```diff
@@ -0,0 +1,169 @@
+package worker
```

Comment:
Renamed file. `DigestEmailBatchJob`, `DigestEmailBatchJobSuccess`,  `DigestEmailJob` are the only new functions